### PR TITLE
Add watchOS platform to Crashlytics

### PR DESCRIPTION
**Pull request by judyliu14**

### CODE REVIEW

**Review by morganchen12 — COMMENTED** [review]
Can you add a method for crashing the app into the app UI?

**Comment by judyliu14** [comment]
> Can you add a method for crashing the app into the app UI?

to clarify, when the user taps on the crash button, it will lead to another page indicating that the app has crashed?

**Comment by morganchen12** [comment]
Sorry for the slow review--it's unclear to me from the code if there's any mechanism for triggering a crash, such as the one you added [here](https://github.com/firebase/quickstart-ios/pull/1193/files#diff-038b85d623e38f01f7abbd982dea0570c5200cf8edbe3d36c430c7da9c3f2b09R22).

**Review by morganchen12 — APPROVED** [review]
LGTM on green CI.

**Comment by morganchen12** [comment]
Looks like the GoogleService-Info.plist needs to be added to all targets:

```
error: Build input file cannot be found: '/Users/runner/Downloads/crashlytics plist/GoogleService-Info.plist' (in target 'CrashlyticsSwiftUIExample (iOS)' from project 'CrashlyticsExample')
```

File: crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -42,10 +96,23 @@
...
+		C745941126BDDD6200153AE3 /* ComplicationController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = ComplicationController.swift; sourceTree = "<group>"; };
+		C745941326BDDD6300153AE3 /* Assets.xcassets */ = {isa = PBXFileReference; lastKnownFileType = folder.assetcatalog; path = Assets.xcassets; sourceTree = "<group>"; };
+		C745941626BDDD6300153AE3 /* Preview Assets.xcassets */ = {isa = PBXFileReference; lastKnownFileType = folder.assetcatalog; path = "Preview Assets.xcassets"; sourceTree = "<group>"; };
+		C745941826BDDD6300153AE3 /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
+		C745941926BDDD6300153AE3 /* PushNotificationPayload.apns */ = {isa = PBXFileReference; lastKnownFileType = text; path = PushNotificationPayload.apns; sourceTree = "<group>"; };
+		C745942C26BDF32400153AE3 /* GoogleService-Info.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; name = "GoogleService-Info.plist"; path = "../../../../Downloads/crashlytics plist/GoogleService-Info.plist"; sourceTree = "<group>"; };
```
Comment (paulb777) [inline crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj:110]: This line is a problem

**Review by paulb777 — APPROVED** [review]

**Pull request merged**